pynumdiff/utils/evaluate.py

```diff
@@ -86,18 +86,16 @@ def robust_rme(u, v, padding=0, M=6):
     :param np.array[float] v: e.g. estimated smoothed signal, reconstructed from derivative
     :param int padding: number of snapshots on either side of the array to ignore when calculating
         the metric. If :code:`'auto'`, defaults to 2.5% of the size of inputs
-    :param float M: Huber loss parameter in units of ~1.4*mean absolute deviation, intended to approximate
-        standard deviation robustly.
+    :param float M: Huber loss parameter in units of ~1.4*mean absolute deviation of population of residual
+        errors, intended to approximate standard deviation robustly.
```

Comment:
In TVR I can't actually find sigma as the scatter of the `y-x` residuals population, because `y` is a cvxpy variable without a set value, so I have to use a different scatter measure to scale the Huber $M$. This got me thinking I should be extra clear about the units $M$ is in in various places throughout the repo.